game/scripts/npc/abilities/electrician_battery_powered.txt

```diff
@@ -0,0 +1,17 @@
+"DOTAAbilities"
+{
+  //=================================================================================================================
+  // Electrician: Battery Powered
+  //=================================================================================================================
+  "electrician_battery_powered"
+  {
+    // General
+    //-------------------------------------------------------------------------------------------------------------
+    "BaseClass"                                           "ability_lua"
+    "AbilityBehavior"                                     "DOTA_ABILITY_BEHAVIOR_PASSIVE | DOTA_ABILITY_BEHAVIOR_NOT_LEARNABLE"
+    "ScriptFile"                                          "abilities/electrician/electrician_battery_powered.lua"
+    "RequiredLevel"                                       "-1"
```

Comment:
You can take a look how I did custom innate for DK or Underlord. Maybe it's simpler, I am not saying this method is wrong, just using Spawn method avoids using KVs as much as possible.